Lexicon.md

```diff
@@ -4,7 +4,8 @@
 * __Bit String__: A sequence of __Bits__ of arbitrary length $$ B^n := (b_k)_{k=0}^n , \  k \in \mathbb{N} $$.
 * __Vulcan Set__: The set of all __Bit Strings__ of infinite length $$ \mathbb{V} := \{ B_0^{\infty}, B_1^{\infty}, \dots, B_{\infty}^{\infty} \} , \ B_i^{\infty} \neq B_j^{\infty} $$.
 * __Information__: Anything that can be encoded into a __Bit String__.
-* __Agent__: Any system that is best characterized as acting according to beliefs.
-* __Agent Boundary__: The surface that separates an __Agent__ from its environment.
-* __Observation__: Information that crosses the __Agent Boundary__, entering the __Agent__.
-* __Action__: Information that crosses the __Agent Boundary__, exiting the __Agent__.
+* __Environment__: __Information__ that is defined relative to a particular __Agent__. It is seperated from an __Agent__ by an __Agent Boundary__, and encompasses all content needed to process any relevant __Actions__ and __Observations__.
+* __Agent__: __Information__ that interacts with an __Environment__. It can consume zero or more __Observations__, and perform zero or more __Actions__. The decision of which __Actions__ to perform is made based on the __Agent's__ beliefs.
```

Comment:
I left the original definition for "acting according to beliefs." Was thinking we could revisit this in a different pull request if needed.

Reply:
You've defined an Agent as Information. I think of it more as a Process, i.e. dynamic. Is it worth distinguishing?

Reply:
My reasoning was that any process could still be encoded into a bit string, and thus be information. We could stamp this with a "pending more lexemes" and revise it once we have more definitions to work with.

Reply:
Agreed, the difference between a program (process) and data (bit string) is largely in the eye of the beholder. We should create a new issue to elaborate later.